client/src/components/commonComponents/Checkbox/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const Checkbox = ({ id, interest, checked = false, onClick }) => (
```

Comment:
this component is a generic checkbox. It is not tied solely to interests. Pls rename `interest` to `value` and all of the CSS class names to start with `checkbox` instead of `interest`.

Reply:
Also, pls utilize defaultProps to initialize the `checked` property and change its propType to not required.